### PR TITLE
Docker image from php:7.4-apache

### PR DESCRIPTION
**Pull request by ook**
Reference: #30 

Provide a functional Dockerfile on php:7-4 with apache.

Exposed port: 80

Readme.md updated with instructions.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ADD docker_files/vhost.conf /etc/apache2/sites-available/000-default.conf
+ADD docker_files/apache.conf /etc/apache2/conf-available/z-app.conf
+ADD docker_files/php.ini /usr/local/etc/php/conf.d/app.ini
+RUN a2enmod rewrite && a2enconf z-app 
+COPY docker_files/errors /errors
+
+COPY index.php /app/index.php
+COPY Cakefile /app/Cakefile
+COPY htaccess.example /app/.htaccess
+RUN sed -i'' -e 's/RewriteBase \/edith\//RewriteBase \//' .htaccess
```
Comment (sunny) [inline Dockerfile:14]: I think we might as well just update the example htaccess, actually.

File: docker_files/errors/403.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Erreur 403 (Interdit)</title>
```
Comment (sunny) [inline docker_files/errors/403.html:7]: Ideally this would stay in english and perhaps with a much more minimal styling that doesn't rely on CDNs as much.

**Review by sunny — COMMENTED** [review]
❤️ ❤️ ❤️ ❤️ ❤️ 

Beware that a `.DS_Store` got through! You should add them to your personal global `~/.gitignore`.

File: docker_files/errors/403.html (hunk 2 above)
Comment (ook) [reply docker_files/errors/403.html:7]: Sure, I borrow those files from a tutorial, will translate all of this

File: Dockerfile (hunk 1 above)
Comment (ook) [reply Dockerfile:14]: As you wish

**Review by sunny — APPROVED** [review]

**Pull request merged**